hooks/queries/useProgramVersionQuery.ts

```diff
@@ -0,0 +1,29 @@
+import { getGovernanceProgramVersion } from '@solana/spl-governance'
+import { PublicKey } from '@solana/web3.js'
+import { useQuery } from '@tanstack/react-query'
+import useWalletStore from 'stores/useWalletStore'
+
+export const programVersionQueryKeys = {
```

Comment:
in `react-query` we manage caching using `queryKey`s. 
See https://tanstack.com/query/v4/docs/guides/query-keys

I personally like to use this factory pattern for `queryKey`s in order to eliminate surface area for typos, but the usefulness of this is not so evident here given that my `queryKey` only gets used once.